research/object_detection/core/preprocessor.py

```diff
@@ -876,8 +870,7 @@ def random_adjust_brightness(image,
         preprocessor_cache.PreprocessorCache.ADJUST_BRIGHTNESS,
         preprocess_vars_cache)
 
-    image = tf.image.adjust_brightness(image, delta)
-    image = tf.clip_by_value(image, clip_value_min=0.0, clip_value_max=1.0)
+    image = tf.image.adjust_brightness(image / 255, delta) * 255
```

Comment:
Please document that image pixel values should be in [0,255].

Reply:
Same with all other functions.

Reply:
@drauh
hi，as data augamention functions needs input pixels to be [0,1] ,but the image pixel datas extracted from tf record files are between [0,255], do you know when does the ssd model precess this?I can't find the it , another thing I'm sure that the values outputs after  data augamention functions are still between [0,255]

Reply:
@liangxiao05  just before images are feeded to the network [here](https://github.com/tensorflow/models/blob/master/research/object_detection/models/ssd_mobilenet_v1_feature_extractor.py#L85)

Reply:
@drauh Right, are all the tests passing? If not, please also update preprocessor_test.py in this PR.

Reply:
@drauh no,it's not here .The augmentation functions are used before this 'preprocess' function. More ,this 'preprocess' function outputs are [-1,1] not [0, 1]  as  the augmentation functions required.

Reply:
Sorry I misunderstood you, I should have said the conversion doesn't takes place and that was the problem.
But augmentation functions doesn't fundamentally require input to be in [0,1]. What is assumed by tf.image.adjust_* ops is that if the type is a float the range should be [0,1] and [0,255] if uint8.

One solution would be to replace the function [tf.to_float](https://github.com/tensorflow/models/blob/master/research/object_detection/trainer.py#L65) by convert_image_dtype but every preprocess functions should be changed, for instance / 255 should be removed [here](https://github.com/tensorflow/models/blob/master/research/object_detection/models/ssd_mobilenet_v1_feature_extractor.py#L85).
An other solution would be to remove this conversion, letting images to be in uint8 while doing data augmentions and then casting them to float just before feeding them to detection_model.preprocess function.